zerver/views/user_settings.py

```diff
@@ -208,6 +208,7 @@ def json_change_settings(
         default=None,
     ),
     starred_message_counts: Optional[bool] = REQ(json_validator=check_bool, default=None),
+    receives_typing_notifications: Optional[bool] = REQ(json_validator=check_bool, default=None),
```

Comment:
If we aren't sorting these parameters, it would probably be best to put this next to the others related to typing notifications. But maybe it's worth just doing a follow-up PR to sort the parameters both here and in `zerver/views/realm.py`; one less thing to make decisions about will be helpful.